### PR TITLE
added 60 nouns to the urban.txt file

### PR DESCRIPTION
**Pull request by ancasimon**

### CODE REVIEW

File: name-segments/nouns/urban.txt [hunk 1]
```diff
@@ -7,4 +7,68 @@ market
...
+metal
+tar
+noise
+clamour
+clatter
+car
+pedal
+light
+queue
+
```
Comment (ianfhunter) [inline name-segments/nouns/urban.txt:71]: No trailing spaces

File: name-segments/nouns/urban.txt [hunk 2]
```diff
@@ -7,4 +7,68 @@ market
 city
 castle
 dragon
-fountain
\ No newline at end of file
+fountain
+bus
```
Comment (ianfhunter) [inline name-segments/nouns/urban.txt:11]:
```suggestion
```

File: name-segments/nouns/urban.txt [hunk 3]
```diff
@@ -7,4 +7,68 @@ market
...
+metro
```
Comment (ianfhunter) [inline name-segments/nouns/urban.txt:12]:
```suggestion
```

File: name-segments/nouns/urban.txt [hunk 4]
```diff
@@ -7,4 +7,68 @@ market
...
+statue
+plaza
+station
+town
+burg
+stage
+rail
```
Comment (ianfhunter) [inline name-segments/nouns/urban.txt:19]:
```suggestion
```

File: name-segments/nouns/urban.txt [hunk 5]
```diff
@@ -7,4 +7,68 @@ market
...
+transit
+way
+mass
+tube
```
Comment (ianfhunter) [inline name-segments/nouns/urban.txt:23]:
```suggestion
```

File: name-segments/nouns/urban.txt [hunk 6]
```diff
@@ -7,4 +7,68 @@ market
...
+bus
+metro
+statue
+plaza
+station
+town
+burg
+stage
+rail
+transit
```
Comment (ianfhunter) [inline name-segments/nouns/urban.txt:20]:
```suggestion
```

**Review by ianfhunter — CHANGES_REQUESTED** [review]
Can you remove the final lines?

**Comment by ancasimon** [comment]
> Can you remove the final lines?

So sorry about that - you are referring to the empty returns I left at the end of the list, right? Done! Please let me know if you would like to see any other changes! Did the same in the mountain adj file too! 😄  @ianfhunter

**Comment by ianfhunter** [comment]
I made a commit to remove some words that were too modern for a fantasy setting.
You should be able to get these changes by doing `git pull` and it'll hopefully solve your issue

**Comment by ancasimon** [comment]
> I made a commit to remove some words that were too modern for a fantasy setting.
> You should be able to get these changes by doing `git pull` and it'll hopefully solve your issue

Done! Thanks for the suggestion @ianfhunter  Please let me know if you need any other changes before merging it!

**Pull request merged**